### PR TITLE
feat(docs): update README of Card (then -> than)

### PR DESCRIPTION
**Pull request by jozef-mikusinec-kiwicom**
Just a simple change in the docs
 Storybook: https://orbit-mainframev-jozef-mikusinec-kiwicom-patch-1.surge.sh

### CODE REVIEW

**Comment by mainframev** [comment]
@jozef-mikusinec-kiwicom thanks for contribution :) can you reword the commit to "docs(Card): update README". It's not a feature :)

**Comment by jozef-mikusinec-kiwicom** [comment]
Sure, reworded

File: packages/orbit-components/src/Card/README.md [hunk 1]
```diff
@@ -18,20 +18,20 @@ After adding import into your project you can use it simply like:
...
+| actions     | `React.Node`                 |         | Optional prop for Action components in header of Card                                                                |
+| children    | `React.Node`                 |         | The content of the Card. You can use only [CardSection](#cardsection)                                                |
+| dataTest    | `string`                     |         | Optional prop for testing purposes                                                                                   |
+| id          | `string`                     |         | Set `id` for `Card`                                                                                                  |
+| description | `React.Node`                 |         | The description of the Card                                                                                          |
+| header      | `React.Node`                 |         | The header of the Card. Useful when you need a different layout than the combination of eg `title` and `description` |
```
Comment (mvidalgarcia) [inline packages/orbit-components/src/Card/README.md:28]:
```suggestion
| header      | `React.Node`                 |         | The header of the Card. Useful when you need a different layout than the combination of e.g. `title` and `description` properties. |
```

For the sake of consistency.

Comment (jozef-mikusinec-kiwicom) [reply packages/orbit-components/src/Card/README.md]: Added the dots

**Review by mvidalgarcia — APPROVED** [review]

**Pull request merged**